docs/circuit_cutting/explanation/index.rst

```diff
@@ -1,5 +1,46 @@
+
+Quasiprobability Decomposition (QPD)
+------------------------------------
+Quasiprobability decomposition is a technique which can be used to simulate noise-free quantum gates using only noisy, local operations (LO). This is often referred to as circuit cutting. One could cut a non-local gate and simulate it using only local operations. This is referred to as a "gate cut" or "space-like cut". One could also imagine cutting a single-qubit identity gate and simulating it using local operations on either side of the cut. The other side of the cut would be simulated by introducing a new qubit into the circuit and moving remaining operations after the cut identity gate to the new qubit. This is referred to as a "wire cut" or "time-like cut". The cost of conducting these techniques is an exponential sampling overhead. If no real-time classical communication is available between qubits of the cut gate or wire, gate cuts incur a sampling overhead of O(9\ :sup:`n`), and wire cuts incur a sampling overhead of O(16\ :sup:`n`), where n is the total number of cuts. If real-time communication is available (i.e. dynamic circuits), the sampling overhead for both gate and wire cuts may be reduced to O(4\ :sup:`n`) [`1 <https://arxiv.org/abs/2205.00016>`_] [`3 <https://arxiv.org/abs/2302.03366>`_]; however, support for circuit cutting with classical communication (LOCC) is not yet supported in CKT.
+
+For more detailed information on this technique, refer to the paper, Error mitigation for short-depth quantum circuits [`4 <https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.119.180509>`_].
+
+Key terms
+-----------------
+* subcircuits: The set of circuits resulting from cutting gates in a `QuantumCircuit` and then separating the disconnected qubit subsets into smaller circuits. These circuits contain `SingleQubitQPDGate`\ s and will be used to instantiate each unique subexperiment.
+
+* subexperiments: A term used to describe the unique circuit samples associated with a subcircuit. These circuits have had their `BaseQPDGate`\ s decomposed into local Qiskit gates and measurements. Subexperiments are the circuits sent to the backend for execution.
+
+* decompose: We try to honor the Qiskit notion of "decompose" in the documentation and API, which loosely means transforming a gate into a less-abstracted representation. *Ocassionally*, we may use the term "decompose" to refer to the act of inserting `BaseQPDGate` instances into quantum circuits as "decomposing" a gate or wire; however, we try to use terms like "partition" and "cut" when referring to this to avoid ambiguity with Qiskit language.
+
+Current limitations
+-------------------
+* QPD-based wire cutting available no sooner than CKT v0.3.0. The `cutqc <https://qiskit-extensions.github.io/circuit-knitting-toolbox/circuit_cutting/cutqc/index.htmlpackage>`_ package may be used for wire cutting in the meantime.
+* `PauliList` is the only supported observable format until no sooner than CKT v.0.3.0
```

Comment:
```suggestion
* ``PauliList`` is the only supported observable format until no sooner than CKT v0.3.0.
```
double backticks since this is rst, plus a small tweak